modules/apps_api/app/controllers/apps_api/v0/directory_controller.rb

```diff
@@ -9,18 +9,22 @@ class DirectoryController < ApplicationController
       skip_before_action(:authenticate)
 
       def index
-        apps = DirectoryApplication.all
         render json: {
-          data: apps
+          data: DirectoryApplication.all
         }
       end
 
       def scopes
         directory_service = Okta::DirectoryService.new
         parsed_scopes = directory_service.scopes(params[:category])
-        render json: {
-          data: parsed_scopes
-        }
+        if parsed_scopes.any? do
+          render json: {
+            data: parsed_scopes
+          }
+           rescue
+             head :no_content
+        end
+        end
```

Comment:
nit: can't say I've seen an "if do" block like this. might just be my unfamiliarity with it, but it was a bit difficult for me to parse. id maybe recommend having a private method that does the render and handling the rescue in there?
```
def scopes
  directory_service = Okta::DirectoryService.new
  parsed_scopes = directory_service.scopes(params[:category])
  
  render_with_parsed_scopes(parsed_scopes) if parsed_scopes.any?
end

private

def render_with_parsed_scopes(parsed_scopes)
  render json: {
    data: parsed_scopes
  }
rescue StandardError
  head :no_content
end
```

but messing around with the syntax you have here and it seems completely valid. just tossing my 2 cents in :+1: